app/templates/agenda.html

```diff
@@ -44,7 +44,11 @@ <h1>{{ start_date.strftime("%d/%m/%Y") }} - {{end_date.strftime("%d/%m/%Y") }}</
     {% for events_date, events_list in events.items() %}
     <div class="shown_dates">{{ events_date.strftime("%d/%m/%Y") }}</div>
         {% for event in events_list %}
-        <div class="event_line"> {{ event[0].start.time().strftime("%H:%M") }} - <a href="/event/view/{{ event[0].id }}">{{ event[0].title }}</a> , duration: {{ event[1] }}</div>
+        <div class="event_line">
+            {% set availability = 'Busy' if event[0].availability == True else 'Free' %}
```

Comment:
Can we be sure there is at least 1 element in `event`?

Reply:
AFAIU, the BE sends a list of [(event, event_duration)].

IMO we can simplify this and pass only list of events, will do in other feature.
Thanks!

Reply:
Sure NP

Reply:
@yammesicka, 
In https://github.com/PythonFreeCourse/calendar/pull/298 the structure is simplified , thanks.